.env.railway

```
@@ -0,0 +1,38 @@
+# Entorno de la aplicación
+APP_ENV=test
+APP_DEBUG=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_DEBUG key should go before the APP_ENV key

(UnorderedKey)